app/assets/stylesheets/settings/_print.scss

```diff
@@ -1,11 +1,17 @@
 @media print {
   @page {
-    size: A4 portrait;
+    size: A4;
+    // margin: 0;
   }
 
-  body {
-    width: 300mm !important;
-    height: 99%;
+  html, body {
+    width: 300mm;
+    margin: 0px 0px 0px 0px; 
```

Comment:
This is the same as margin: 0;
No need to assign all, no need to assing 'px' either.